pkg/utils/test_util.go

```diff
@@ -136,3 +142,32 @@ func (matcher AlreadyExistMatcher) FailureMessage(actual interface{}) (message s
 func (matcher AlreadyExistMatcher) NegatedFailureMessage(actual interface{}) (message string) {
 	return format.Message(actual, "not to be already exist")
 }
+
+// MockHandle framework.Handle interface for set up the plugin.
+type MockHandle struct {
```

Comment:
All the test util functions need to be moved to the test/utils/ path. We should not mix the test util with the non test utils in one package.